knowage-core/src/main/java/it/eng/spagobi/api/AbstractDocumentResource.java

```diff
@@ -50,7 +50,7 @@ public abstract class AbstractDocumentResource extends AbstractSpagoBIResource {
 
 	private static final Logger LOGGER = LogManager.getLogger(AbstractDocumentResource.class);
 
-	public Response insertDocument(String body) {
+	protected Response insertDocument(String body) {
```

Comment:
If possible, I would prefer private but this is an abstract class, we need a check.

Reply:
This one can't be private, it is being used in DocumentResource.java (two files, one of them is v2).